### ISSUE TITLE
Correct validation/test set split

### ISSUE THREAD

**Issue by dbeinhauer** [dbeinhauer/mcs-source#103]
Currently, our implementation of validation/test set split is not ideal. During our training we use only 10 batches (validation set) for evaluation of our model performance. At the end, we should ideally take only the rest of them to test the actual performance on not seen data. Additionally, we should select these sets kind of shuffled based on our different datasets (when I would have multiple different model subsets).

**Comment by dbeinhauer** [comment]
Not really a bug. One just needs to be aware that while running grid search we need to check only the preliminary cc_norm not the final (that is on the evaluaion and test dataset).
